debian/kolibri-server.init

```diff
@@ -31,11 +31,11 @@ PIDFILE_UWSGI=/var/run/$NAME/uwsgi.pid
 
 KOLIBRI_USER_HOME="$(getent passwd $KOLIBRI_USER | awk -F ':' '{print $6}')"
 
-# I don't like this, what if the user decides to set KOLIBRI_HOME somewhere else?
-DAEMON_HOME="$KOLIBRI_USER_HOME/.kolibri"
+# Expands ~ in the KOLIBRI_HOME to the KOLIBRI_USER's HOME
+KOLIBRI_HOME=${KOLIBRI_HOME//\~/"$KOLIBRI_USER_HOME"}
```

Comment:
why?

Reply:
I did this over a month ago, so would have to go back and revisit why there was a need to expand the `~` path here.

Reply:
Sorry for not getting back on this. I recall the issue like this: I tried to put "~/.kolibri_alternative" as `KOLIBRI_HOME` in `/etc/kolibri/conf.d/some-file` and thought it was clever because it could locate the Kolibri home directory relative to whichever user owns the process.

But this failed because something needed the absolute path and this seemed like a good place to expand it.

I would think it's better to put this logic in `/etc/default/kolibri` (and get rid of this file as a separate issue), so I'll remove it from this PR and implement it in the `kolibri` package.

Reply:
yep, but in any case I don't think `~/ `should be allowed in a config file.